src/main/java/com/mraof/minestuck/item/weapon/aspect/BreathAOEWeaponItem.java

```diff
@@ -0,0 +1,59 @@
+package com.mraof.minestuck.item.weapon.aspect;
+
+import com.mraof.minestuck.item.weapon.KnockbackWeaponItem;
+import com.mraof.minestuck.item.weapon.MSToolType;
+import com.mraof.minestuck.player.EnumAspect;
+import com.mraof.minestuck.player.Title;
+import com.mraof.minestuck.world.storage.PlayerSavedData;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.ServerPlayerEntity;
+import net.minecraft.item.IItemTier;
+import net.minecraft.item.ItemStack;
+import net.minecraft.particles.ParticleTypes;
+import net.minecraft.potion.EffectInstance;
+import net.minecraft.potion.Effects;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvents;
+import net.minecraft.util.math.AxisAlignedBB;
+
+import java.util.List;
+
+public class BreathAOEWeaponItem extends KnockbackWeaponItem
```

Comment:
The aspect is passed as an argument, so I'd think that this is better called a levitation-aoe-weapon